scene/main/timer.cpp

```diff
@@ -158,6 +158,13 @@ double Timer::get_time_left() const {
 	return time_left > 0 ? time_left : 0;
 }
 
+void Timer::set_time_left(double p_time_left) {
+	if (!processing) {
```

Comment:
Nitpick: it is godot style to only put the ERR_FAIL macro and not the if () statement. The macro contains the if (!processing) check internally, so the if statement is redundant